### DISCUSSION TITLE
on:disconnect event doesn't provide disconnected from

### DISCUSSION THREAD

**Discussion by TomFaulkner** [open-source-labs/Svelvet#434] [General]
The event doesn't provide info on what the node was disconnected from. Is there a method to gather that information?

Also, it doesn't seem like there is a way to disconnect from an anchor with multiple connections.

The library is working great for my needs so far, aside from these two things. Thanks for the hard work.

**Comment by Felipe7575** [comment]
Have you found any way to solve this? Im having the same problem.
